packages/aws-s3-workflows/src/s3_utils.test.ts

```diff
@@ -1,350 +1,350 @@
-    const myFileRec = await TestUserFileTable.chooseFileRecord(myFile);
-    const wfhandle = await DBOS.startWorkflow(uploadPWF)(myFileRec, 60, { contentType: 'text/plain' });
-    //    Get the presigned post
-    const ppost = await DBOS.getEvent<PresignedPost>(wfhandle.workflowID, 'uploadkey');
-    //    Upload to the URL
-    try {
-      const res = await uploadToS3(ppost!, './src/s3_utils.test.ts');
-      expect(res.status.toString()[0]).toBe('2');
-    } catch (e) {
-      // You do NOT want to accidentally serialize an AxiosError - they don't!
-      console.log('Caught something awful!', e);
-      expect(e).toBeUndefined();
-    }
-    //    Notify WF
-    await DBOS.send<boolean>(wfhandle.workflowID, true, 'uploadfinish');
-
-    //    Wait for WF complete
-    const _myFileRecord = await wfhandle.getResult();
-
-    // Get the file out of DBOS (using a signed URL)
-    const myurl = await getS3KeyUrl(myFileRec.key, 60);
-    expect(myurl).not.toBeNull();
-    // Get the file contents out of S3
-    await downloadFromS3(myurl, './deleteme.xxx');
-    expect(fs.existsSync('./deleteme.xxx')).toBeTruthy();
-    fs.rmSync('./deleteme.xxx');
-
-    // Delete the file contents out of DBOS (using the table index)
-    const dfhandle = await deleteWF(myFileRec);
-    expect(dfhandle).toBeDefined();
-  }, 10000);
-
-  async function getS3KeyContents(key: string) {
-    return (
-      await s3client!.send(
-        new GetObjectCommand({
-          Bucket: s3bucket!,
-          Key: key,
-        }),
-      )
-    ).Body?.transformToString();
-  }
-
-  async function getS3KeyUrl(key: string, expirationSecs: number) {
-    const getObjectCommand = new GetObjectCommand({
-      Bucket: s3bucket!,
-      Key: key,
-    });
-
-    const presignedUrl = await getSignedUrl(s3client!, getObjectCommand, { expiresIn: expirationSecs });
-    return presignedUrl;
-  }
-
-  async function uploadToS3(presignedPostData: PresignedPost, filePath: string) {
-    const formData = new FormData();
-
-    // Append all the fields from the presigned post data
-    Object.keys(presignedPostData.fields).forEach((key) => {
-      formData.append(key, presignedPostData.fields[key]);
-    });
-
-    // Append the file you want to upload
-    const fileStream = fs.createReadStream(filePath);
-    formData.append('file', fileStream);
-
-    return await axios.post(presignedPostData.url, formData);
-  }
-
-  async function downloadFromS3(presignedGetUrl: string, outputPath: string) {
-    const response: AxiosResponse<Readable> = await axios.get(presignedGetUrl, {
-      responseType: 'stream', // Important to handle large files
-    });
-
-    // Use a write stream to save the file to the desired path
-    const writer = fs.createWriteStream(outputPath);
-    response.data.pipe(writer);
-
-    return new Promise<void>((resolve, reject) => {
-      writer.on('finish', resolve);
-      writer.on('error', reject);
-    });
-  }
-});
+// import {
```

Comment:
This test (and associated knexfile) are still on TSv2 and I can't run them locally. Commenting out for now, we can decide if we want to keep this package or remove it.